hack/make-rules/test-cmd-util.sh

```diff
@@ -4861,6 +4861,10 @@ runTests() {
     kube::log::status "Creating namespace ${ns_name}"
     kubectl create namespace "${ns_name}"
     kubectl config set-context "${CONTEXT}" --namespace="${ns_name}"
+
+    # convenient spot to check for namespace kinds being set.
+    # TODO before merge, I'll make a real test
```

Comment:
...and add a TODO that we probably need to do this for embedded types that don't happen to be list items, if we have any :(

Reply:
> ...and add a TODO that we probably need to do this for embedded types that don't happen to be list items, if we have any :(

Not as bad as you think.  My test failures here indicate that we actually check this in fuzz tests, so I update the fuzz tests and we get full coverage.